src/fu-util.c

```diff
@@ -2424,6 +2427,63 @@ fu_util_upload_security_build_metadata (JsonBuilder *builder, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_upload_security_build_kernel_metadata (JsonBuilder *builder, GError **error)
+{
+	gsize bufsz = 0;
+	g_autofree gchar *buf = NULL;
+#ifdef HAVE_UTSNAME_H
+	struct utsname uname_tmp;
+#endif
+	const gchar *ignore[] = {
+		"",
+		"BOOT_IMAGE",
+		"console",
+		"quiet",
+		"quiet",
```

Comment:
double quiet

Reply:
Fixed, thanks. Twitter replies still coming in, so I might add a few more things here.